python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/spanning_tree.schema.yml

```
@@ -107,3 +107,19 @@ keys:
               type: int
               convert_types:
                 - str
+      spanning_tree_port_id_allocation_port_channel_range:
```

For documentation purpose here is what the config looks like today:


```
site3-wan1(config)#spanning-tree port?
port-id  portchannel

site3-wan1(config)#spanning-tree port-id ?
  allocation  Enable internal STP port-id allocation scheme

site3-wan1(config)#spanning-tree port-id allocation ?
  port-channel  Reserve port-ids for port-channels

site3-wan1(config)#spanning-tree port-id allocation port-channel ?
  range  Specify range of port-ids to reserve for port-channels

site3-wan1(config)#spanning-tree port-id allocation port-channel range ?
  <1-2048>  Specify minimum value for reserved range

site3-wan1(config)#spanning-tree port-id allocation port-channel range 666 ?
  <1-2048>  Specify maximum value for reserved range
```

I would suggest

```suggestion
port_id_allocation_port_channel_range:
```

---

updated